Dockerfile

```diff
@@ -5,5 +5,17 @@
 
 FROM alpine:3.2
 RUN apk add -U ca-certificates git openssh curl perl && rm -rf /var/cache/apk/*
+
+ENV LFS_VERSION 1.0.2
+RUN apk add --update --virtual build-dependencies curl && \
+    curl -sLO https://github.com/github/git-lfs/releases/download/v${LFS_VERSION}/git-lfs-linux-amd64-${LFS_VERSION}.tar.gz && \
```

Comment:
@bradrydzewski You're right we can remove those lines since curl is already available.

`build-dependencies` is the name of the virtual package created with `--virtual`. It's particularly useful to remove group of installed packages.

It's irrelevant here as we do not need to remove curl. Worse, it even removes it whereas we wanted to keep it 👎 

Sorry for the mistakes, I wasn't yet aware of Alpine's `apk` specifics 😕